src/devices/BuildHat/BuildHat.csproj

```diff
@@ -0,0 +1,49 @@
+<Project Sdk="Microsoft.NET.Sdk">
+   <PropertyGroup>
+    <TargetFrameworks>$(DefaultBindingTfms)</TargetFrameworks>
+    <EnableDefaultItems>false</EnableDefaultItems>
+  </PropertyGroup>
+
+   <ItemGroup>
+     <PackageReference Include="System.IO.Ports" Version="$(SystemIOPortsPackageVersion)" />
+     <Compile Include="Brick.cs" />
+     <Compile Include="Models\BuildHatInformation.cs" />
+     <Compile Include="Models\CombiModes.cs" />
+     <Compile Include="Models\LedColor.cs" />
+     <Compile Include="Models\LedMode.cs" />
+     <Compile Include="Models\MinimumMaximumValues.cs" />
+     <Compile Include="Models\ModeDetail.cs" />
+     <Compile Include="Models\PositionWay.cs" />
+     <Compile Include="Models\RecommendedPid.cs" />
+     <Compile Include="Models\SensorPort.cs" />
+     <Compile Include="Models\SensorType.cs" />
+     <Compile Include="Models\TypeValues.cs" />
+     <Compile Include="Motors\ActiveMotor.cs" />
+     <Compile Include="Motors\IMotor.cs" />
+     <Compile Include="Motors\PassiveMotor.cs" />
+     <Compile Include="Sensors\ActiveSensor.cs" />
+     <Compile Include="Sensors\ButtonSensor.cs" />
+     <Compile Include="Sensors\ColorAndDistanceSensor.cs" />
+     <Compile Include="Sensors\ColorLightMatrix.cs" />
+     <Compile Include="Sensors\ColorSensor.cs" />
+     <Compile Include="Sensors\UltrasonicDistanceSensor.cs" />
+     <Compile Include="Sensors\ForceSensor.cs" />
+     <Compile Include="Sensors\Sensor.cs" />
+     <Compile Include="Sensors\PassiveLight.cs" />
+     <Compile Include="Sensors\WeDoDistanceSensor.cs" />
+     <Compile Include="Sensors\WeDoTiltSensor.cs" />
+   </ItemGroup>
+
+   <ItemGroup>
+     <None Include="data\firmware.bin">
```

Comment:
I would embed these as resources, otherwise we have to do special tricks to get them deployed correctly during package consumption.

Reply:
yes, I can do that. That's a good catch!